### PR TITLE
Document HTMLElement.enterKeyHint

### PR DESCRIPTION
**Pull request by Elchi3**
This is the API part of `enterkeyhint`. Following up from the HTML part in https://github.com/mdn/content/pull/1106.

Spec: https://html.spec.whatwg.org/multipage/interaction.html#dom-enterkeyhint

Why document this under `HTMLElement`? 
Well, the spec defines it under [`interface mixin ElementContentEditable`](https://html.spec.whatwg.org/multipage/interaction.html#elementcontenteditable) which [`HTMLElement` includes](https://html.spec.whatwg.org/multipage/dom.html#htmlelement). I don't think we want to document the `ElementContentEditable` mixin.

### CODE REVIEW

File: files/en-us/web/api/htmlelement/enterkeyhint/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ <li><code>'previous'</code> typically taking the user to the previous field that will accept text.	</li>
+ <li><code>'search'</code> typically taking the user to the results of searching for the text they have typed.</li>
+ <li><code>'send'</code> typically delivering the text to its target.	</li>
+</ul>
+
+<p>If no <code>enterKeyHint</code> value has been specified or if it was set to a different value than the allowed ones, it will return an empty string.</p>
+
+<h2 id="Examples">Examples</h2>
+<p>Give a virtual keyboard a hint how to label the enter key (might render as <kbd>Send</kbd> and <kbd>Search</kbd>, depending on the OS or the user's language).</p>
+<pre class="brush: js notranslate">
```
Comment (peterbe) [inline files/en-us/web/api/htmlelement/enterkeyhint/index.html:34]: Just FYI, you don't need to insert the `notranslate` into `<pre>` tags. It gets automatically inserted in build-time.

Comment (Elchi3) [reply files/en-us/web/api/htmlelement/enterkeyhint/index.html:34]: Ah thanks! I just copied this from existing sources. Might be nice cleaning to remove `notranslate` then everywhere at some point.

File: files/en-us/web/api/htmlelement/enterkeyhint/index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - Property
+  - Reference
+  - Keyboard
+---
+<div>{{APIRef("HTML DOM")}}</div>
+
+<p>The <strong><code>enterKeyHint</code></strong> property is an enumerated property defining 
+what action label (or icon) to present for the enter key on virtual keyboards. <br>
+It reflects the <a href="/en-US/docs/Web/HTML/Global_attributes/enterkeyhint"><code>enterkeyhint</code></a>
+HTML global attribute and is an enumerated property only accepting the following values
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmlelement/enterkeyhint/index.html:17]:
```suggestion
HTML global attribute and is an enumerated property, only accepting the following values 
```

**Review by sideshowbarker — DISMISSED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/api/htmlelement/enterkeyhint/index.html (hunk 1 above)
Comment (Elchi3) [reply files/en-us/web/api/htmlelement/enterkeyhint/index.html:34]: Filed https://github.com/mdn/content/issues/1250

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**